dvc/stage/__init__.py

```diff
@@ -55,6 +55,7 @@ def loads_from(cls, repo, path, wdir, data):
                 Stage.PARAM_ALWAYS_CHANGED,
                 Stage.PARAM_MD5,
                 Stage.PARAM_DESC,
+                Stage.PARAM_JOBS,
```

Comment:
Let's not add it, this is too specific for your particular connection and might be undesired for other users.

Reply:
Ah, I see that you don't save it, you just utilize the loading this way. Hm, I guess we could keep it as is :+1: Let me see...

Reply:
Yes, without this, I can't create a stage with the `jobs` member variable. Stage params' saving is in `dvc.schema.py`.

Here we have another two ways to pass this new variable:

1. Pass `jobs` through `Stage.run()`, but more ugly.
2. Add a new set or fill method? something like `fill_stage_outputs` and `fill_stage_dependencies`, more specific.